File: src/android/com/adobe/phonegap/push/utils/ServiceUtils.java
package com.adobe.phonegap.push.utils;

import android.app.NotificationManager;
import android.content.Context;
import android.media.AudioManager;
import android.os.PowerManager;
import android.os.Vibrator;

public class ServiceUtils {

    public static PowerManager getPowerService() {
        return (PowerManager) Globals.applicationContext.getSystemService(Context.POWER_SERVICE);
    }

    public static NotificationManager getNotificationService() {
        return (NotificationManager) Globals.applicationContext.getSystemService(Context.NOTIFICATION_SERVICE);
    }

    public static Vibrator getVibratorService() {
        return (Vibrator) Globals.applicationContext.getSystemService(Context.VIBRATOR_SERVICE);
    }

    public static AudioManager getAudioService() {
        return (AudioManager) Globals.applicationContext.getSystemService(Context.AUDIO_SERVICE);
    }
}

File: src/android/com/adobe/phonegap/push/utils/ServiceUtils.kt
package com.adobe.phonegap.push.utils

import android.os.PowerManager
import android.app.NotificationManager
import android.content.Context
import android.os.Vibrator
import android.media.AudioManager

object ServiceUtils {
    fun powerService(context: Context): PowerManager {
        return context.getSystemService(Context.POWER_SERVICE) as PowerManager
    }
    fun notificationService(context: Context): NotificationManager {
        return context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
    }
    fun vibratorService(context: Context): Vibrator {
        return context.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
    }
    fun audioService(context: Context): AudioManager {
        return context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
    }
}